fix: hva skjer med en naisjob (#225)
nais/doc: docs/naisjob/README.md

  schedule: "*/1 * * * *"
```

> Behind the scene when you make a `Naisjob`, Kubernetes makes a bunch of different resources for you.

Reviewer: ```suggestion
Behind the scenes, when you make a `Naisjob`, Kubernetes makes a bunch of different resources for you.
```

`Naisjob` is one resource that has been defined by the NAIS team.
When you deploy a `Naisjob` with `spec.schedule`, Kubernetes will create a `CronJob` for you.

> A `Cronjob` is a Kubernetes native resource.

Reviewer: ```suggestion
A [`CronJob`](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/) is a Kubernetes native resource.

Author: dette er også lenket til i starten

> When the `Cronjob`-`schedule` triggers, Kubernetes will create a `Job`-ressource, which creates a `Pod`-resource.

Reviewer: ```suggestion
When the `Cronjob`-`schedule` triggers, Kubernetes will create a [`Job`](https://kubernetes.io/docs/concepts/workloads/controllers/job/)-resource, which creates a [`Pod`](https://kubernetes.io/docs/concepts/workloads/pods/)-resource.
```

Author: Trenger vi å lenke til Job-doc igjen? Er gjort i starten av dokumentet. Pod kan sikkert være fint ja.

  schedule: "*/1 * * * *"
```

Behind the scenes, when you make a `Naisjob`, Kubernetes makes a bunch of different resources for you.
`Naisjob` is one resource that has been defined by the NAIS team.
When you deploy a `Naisjob` with `spec.schedule`, Kubernetes will create a `CronJob` for you.
A `Cronjob` is a Kubernetes native resource.

> When the `Cronjob`-`schedule` triggers, Kubernetes will create a `Job`-ressource, which creates a `Pod`-resource.

Author: ```suggestion
When the `Cronjob`-`schedule` triggers, Kubernetes will create a `Job`-ressource, which creates a [`Pod`](https://kubernetes.io/docs/concepts/workloads/pods/)-resource.
```
